apps/explorer/src/pages/AddressInfoPage/ExportAddressTXsModal.tsx

```diff
@@ -67,7 +76,12 @@ const ExportAddressTXsModal = ({ addressHash, onClose, ...props }: ExportAddress
 
       if (!data) throw 'Something wrong happened while fetching the data.'
 
-      startCSVFileDownload(data, `${addressHash}__${timePeriodValue}__${dayjs().format('DD-MM-YYYY')}`)
+      startCSVFileDownload(
+        data,
+        `${addressHash}__${dayjs(timePeriods[timePeriodValue].from).format(SIMPLE_DATE_FORMAT)}-${dayjs(
```

Comment:
nit: extract `dayjs(timePeriods[timePeriodValue].from).format(SIMPLE_DATE_FORMAT)` into vars to improve readability

Reply:
Done